client/src/js/base/__tests__/RelativeTime.test.js

```diff
@@ -3,73 +3,49 @@ import { RelativeTime } from "../RelativeTime";
 
 describe("<RelativeTime />", () => {
     let props;
-    let wrapper;
-    let expected;
+    let state;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['state' is assigned a value but never used.](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=4770288)